src/components/3d_editor/advanced_geometry/CombinatorialBasisDialog.jsx

```diff
@@ -109,7 +109,7 @@ class CombinatorialBasisDialog extends React.Component {
                                 <IconByName name="shapes.info" />
                             </a>
                         </Typography>
-                        <IconButton color="neutral" onClick={onHide}>
+                        <IconButton color="default" onClick={onHide}>
```

Comment:
should color be "primary" or "secondary"?

Reply:
This wasn't my addition. TB's recent developments should fix this anyway